debian/marquee/README.md

```diff
@@ -3,6 +3,5 @@ marquee hosts simple static sites.
 Repos that deploy to this server:
 - https://github.com/whatwg/books/blob/master/.travis.yml
 - https://github.com/whatwg/figures/blob/master/.travis.yml
-- https://github.com/whatwg/html-differences/blob/master/.travis.yml
```

Comment:
Shall delete that config and archive https://github.com/whatwg/html-differences to make sure that it's never deployed again?

Reply:
Yeah, that's the plan per #100.